src/schemata/definitions/field_schema/file.yml

```
@@ -9,11 +9,7 @@ properties:
     enum: [file]
   mediaType:
     description: |
-      A single or list of accepted [Media Types](https://www.iana.org/assignments/media-types/media-types.xhtml).
+      The media format of the specific file. Kindly see the accepted [Media Types](https://www.iana.org/assignments/media-types/media-types.xhtml).
 
       Note: Not all Media Types out there might be supported.
```

```suggestion
      A [media type](https://www.iana.org/assignments/media-types/media-types.xhtml) is a two-part identifier for file formats and format contents. Note that not all possible media types might be supported.
```
Let's rewrite this a bit to avoid linking "accepted" media types and the immediately pointing out that not all are accepted.

This changes the schema features, so please also open a PR on the server repo.